<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationsListDialog</class>
 <widget class="QDialog" name="StationsListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Stations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="nearButton">
     <property name="text">
      <string>Nearest</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="StationsListView" name="listView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationsListView</class>
   <extends>QListView</extends>
   <header>stationslistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
